src/index.js.flow

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import type { Sink, Scheduler, Source, Disposable } from "most";
+import { Stream, } from "most";
+
+declare export default function multicast<A>(s: Stream<A>): Stream<A>;
+
+declare export class MulticastSource<A> {
+  constructor(source: Source<A>): MulticastSource<A>;
+  run (sink: Sink<A>, scheduler: Scheduler): Disposable<A>;
```

Comment:
Extra space here.  I realize this was probably copied from the TS defs, so we can fix it there too.